### PR TITLE
Need psycopg2-binary to work

### PR DESCRIPTION
**Pull request by ChristianDresel**
The Script don't work without psycopg2-binary

### CODE REVIEW

**Comment by turt2live** [comment]
For which OS does this not work?

**Comment by ChristianDresel** [comment]
Debian 9 stretch minimal new installation

File: docs/postgres.rst [hunk 1]
```diff
@@ -28,6 +28,7 @@ virtual env::
 
     sudo apt-get install libpq-dev
     pip install psycopg2
+    pip install psycopg2-binary
```
Comment (richvdh) [inline docs/postgres.rst]: as I understand it `psycopg2` and `psycopg2-binary` are alternatives - why would we need both?

(and even if we do need both, why do they need to be installed with separate `pip` invocations?)

**Comment by richvdh** [comment]
According to https://pypi.python.org/pypi/psycopg2-binary:

> The binary package is a practical choice for development and testing but in production it is advised to use the package built from sources.

We'll need justification as to why `psycopg2` is unsuitable, beyond "The Script don't work", to go against this advice.

**Comment by richvdh** [comment]
ok, I'm closing this for now. If you can provide any more info, we'd be happy to reopen it.

**Pull request closed without merging**